src/sidebar/components/MarkdownView.tsx

```diff
@@ -19,13 +35,20 @@ export default function MarkdownView({
   markdown,
   classes,
   style,
+  mentions,
 }: MarkdownViewProps) {
   const html = useMemo(
     () => (markdown ? renderMathAndMarkdown(markdown) : ''),
     [markdown],
   );
   const content = useRef<HTMLDivElement | null>(null);
 
+  const hasMentions = !!mentions?.length;
+  const [popoverContent, setPopoverContent] = useState<Mention | string | null>(
+    null,
+  );
```

Comment:
It isn't immediately obvious what the `string` type means here. I would add a comment.

Reply:
We should probably use the `InvalidUsername` type here, which is an alias for `string` and more self-explanatory.